packages/@aws-cdk/aws-s3/lib/bucket.ts

```diff
@@ -985,4 +997,8 @@ class ImportedBucket extends BucketBase {
   private generateDomainName() {
     return `${this.bucketName}.s3.amazonaws.com`;
   }
+
+  private generateBucketWebsiteUrl() {
+    return `${this.bucketName}.s3-website-${new cdk.AwsRegion()}.amazonaws.com`;
```

Comment:
how does this look in a different partition? (i.e. china)

Reply:
That's a.. good question. I wasn't aware (until I found [these docs](https://docs.amazonaws.cn/en_us/aws/latest/userguide/services.html)) that the domain name was different in regions such as China.

Happy to take a look at making sure this works in regions such as China, would you want me to take a look at doing the same for the `generateDomainName()` function above (as that appears to have the same problem), or leave that for a separate issue and PR?